governance/polls/SparkLend Ethereum - Increase WBTC supply cap max parameter to 10k - March 25, 2024.md

```diff
@@ -0,0 +1,55 @@
+---
+title: SparkLend Ethereum - Increase WBTC supply cap max parameter to 10k - March 25, 2024
```

Comment:
:yellow_circle: **minor**: capitals in title

Make consistent with prior poll https://vote.makerdao.com/polling/QmYYoAMe

```suggestion
title: SparkLend Ethereum - Increase WBTC Supply Cap Maximum - March 25, 2024
```

Reply:
capitals in file name if updating